### DISCUSSION TITLE
Hardcore Buckets tweak

### DISCUSSION THREAD

**Discussion by ACGaming** [ACGaming/UniversalTweaks#40] [Tweak Suggestions]
https://gitlab.com/BetterWithMods/1.12/BetterWithMods/-/blob/1.12/src/main/java/betterwithmods/module/hardcore/world/HCBuckets.java

**Comment by ACGaming** [comment]
Implemented in f8032f160e0d2d1397975b76b31b398aa0218abd.

**Comment by Primitive-Human** [comment]
Wanted to make this suggestion, but surprised to find this.
Question though, does it entirely prevent placement of fluids, or does the placed fluid immediately spread so that it cannot be recollected in a bucket?

**Reply by ACGaming** [reply]
It only creates a small puddle which eventually despawns.
